### PR TITLE
Add ping cloud function and healthCheck endpoint

### PR DESCRIPTION
**Pull request by avalmas-programize**
Add ping cloud function (schedule ping for every 5 minutes)
New cloud function env variable is needed (backend_base_url => The base url of the backend api)
Add healthCheck module and controller (used as a simple ping endpoint):
- GET /health-check: Return { status: 200 }

### CODE REVIEW

File: packages/api/cloud-functions/index.ts [hunk 1]
```diff
@@ -80,3 +81,14 @@ exports.scheduledDailyUpdate = functions
       conn.close();
     }
   });
+
+exports.pingService = functions
+  .runWith({ timeoutSeconds: 540 })
```
Comment (ericboucher) [inline packages/api/cloud-functions/index.ts:86]: I don't think we need a 9min timeout for this

File: packages/api/cloud-functions/index.ts [hunk 2]
```diff
@@ -80,3 +81,14 @@ exports.scheduledDailyUpdate = functions
...
+  .pubsub.schedule('*/5 * * * *')
+  .timeZone('America/Los_Angeles')
```
Comment (ericboucher) [inline packages/api/cloud-functions/index.ts:88]: no need for a timezone either

**Comment by ericboucher** [comment]
@avalmas-programize let's make sure you update the README command with the new variable:


export $(grep -v '^#' .env.staging | xargs) && firebase functions:config:set database.url=$DATABASE_URL sofar_api.token=$SOFAR_API_TOKEN ...

**Review by ericboucher — CHANGES_REQUESTED** [review]
See inline comments + add env_variable

File: packages/api/cloud-functions/index.ts (hunk 2 above)
Comment (avalmas-programize) [reply packages/api/cloud-functions/index.ts:88]: Ok

File: packages/api/cloud-functions/index.ts (hunk 1 above)
Comment (avalmas-programize) [reply packages/api/cloud-functions/index.ts:86]: Ok

**Review by ericboucher — APPROVED** [review]

**Pull request merged**